backend/data/src/main/kotlin/io/tolgee/component/machineTranslation/providers/BaiduTranslationProvider.kt

```diff
@@ -30,167 +30,98 @@ class BaiduTranslationProvider(
 
   override val supportedLanguages = arrayOf(
     "yue",
-    "yue-hans",
```

Comment:
No, no... Don't remove it from supported languages. We don't want fallback when not necessary.